ami/manager.go

```diff
@@ -207,3 +207,25 @@ func DeviceStateList(ctx context.Context, client Client, actionID string) ([]Res
 func LoggerRotate(ctx context.Context, client Client, actionID string) (Response, error) {
 	return send(ctx, client, "LoggerRotate", actionID, nil)
 }
+
+// UpdateConfig Updates a config file.
+// Dynamically updates an Asterisk configuration file.
+func UpdateConfig(ctx context.Context, client Client, actionID, srcFilename, dstFilename string, actions []UpdateConfigAction, reload bool) (Response, error) {
```

Comment:
I think that you can move the actions to be the last parameter, for example:
```go
func UpdateConfig(ctx context.Context, client Client, actionID, srcFilename, dstFilename string, 
  reload bool, actions ...UpdateConfigAction) (Response, error) {
   ...

}
```
Being possible to append Config actions like this:
```go
action01 := ami.UpdateConfigAction{}
action02 := ami.UpdateConfigAction{}
UpdateConfig(ctx, client, id, src, dst, false, action01, action02)

```